jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -935,68 +1001,19 @@ ecma_date_to_string (ecma_value_t this_arg, /**< this argument */
   }
   else
   {
-    ECMA_TRY_CATCH (obj_this,
-                    ecma_op_to_object (this_arg),
-                    ret_value);
+    ecma_completion_value_t completion = ecma_op_to_object (this_arg);
```

Comment:
There is still `ToObject` conversion. I don't think we need it, as we execute the block only if `this_arg` contains object with `Date` class name.